dask/dataframe/core.py

```diff
@@ -2801,11 +2832,21 @@ def _maybe_from_pandas(dfs):
     return dfs
 
 
+def hash_shard(df, nparts, index):
+    if index:
+        h = df.index
+    else:
+        h = df
+    h = hash_pandas_object(h) % nparts
+
+    return {i: df.iloc[h == i] for i in range(nparts)}
+
+
 @insert_meta_param_description
 def apply_concat_apply(args, chunk=None, aggregate=None, combine=None,
                        meta=no_default, token=None, split_every=None,
                        chunk_kwargs=None, aggregate_kwargs=None,
-                       combine_kwargs=None, **kwargs):
```

Comment:
is ``split_out`` able to be specified once per ``.agg``?

e.g.

```
ddf.groupby(..).agg(['min', max'])
```

where/how can I specifed it?

maybe
``ddf.groupby(..).agg(['min', max'], split_out=N)`` ? (or *maybe* in ``.groupby`` which would be a default)